app/containers/ImageInput/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ImageInputComponent from 'components/ImageInput';
+
+class ImageInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.fileUpload = React.createRef();
+    this.showFileUpload = this.showFileUpload.bind(this);
```

Comment:
Please avoid using `bind` (it creates a new function every time and slows down the app). You can use `(...params) => this.showFileUpload(...params)` instead.